pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "diff_cover"
-version = "7.1.0"
+version = "7.1.1"
```

Comment:
let me handle this. I just do this when I bump versions. 

This is a feature and non breaking so I would probably go ahead and make this release 7.2.0 anyway